js/app.js

```diff
@@ -0,0 +1,61 @@
+let zombieTypes = {
+    dead: [],
```

Comment:
any chance these arrays could be the simple counters like 
zombieTypes = {
    dead: 0,
...
}